Core/Wrappers/Damages/Damage.cs

```diff
@@ -386,6 +506,15 @@ private static double CalculatePhysicalDamage(this Obj_AI_Base source, Obj_AI_Ba
                         armorPenetrationPercent = 0.3d;
                         break;
                 }
+                if (target is Obj_AI_Minion)
```

Comment:
case TurretType.TierTwo:
                         armorPenetrationPercent = 0.7d;

shouldn't it be 0,75?